src/cmd/singularity/cli/build.go

```diff
@@ -29,23 +31,150 @@ var (
 )
 
 func init() {
-	buildCmd.Flags().SetInterspersed(false)
-	singularityCmd.AddCommand(buildCmd)
-
-	buildCmd.Flags().BoolVarP(&Sandbox, "sandbox", "s", false, "Build image as sandbox format (chroot directory structure)")
-	buildCmd.Flags().StringSliceVar(&Sections, "section", []string{}, "Only run specific section(s) of deffile")
-	buildCmd.Flags().BoolVarP(&Writable, "writable", "w", false, "Build image as writable (SIF with writable internal overlay)")
-	buildCmd.Flags().BoolVarP(&Force, "force", "f", false, "")
-	buildCmd.Flags().BoolVarP(&NoTest, "notest", "T", false, "")
-	buildCmd.Flags().BoolVarP(&Remote, "remote", "r", false, "Build image remotely")
-	buildCmd.Flags().StringVar(&RemoteURL, "remote-url", "localhost:5050", "Specify the URL of the remote builder")
-	buildCmd.Flags().StringVar(&AuthToken, "auth-token", "", "Specify the auth token for the remote builder")
+
+	manHelp := func(c *cobra.Command, args []string) {
```

Comment:
Well, I don't think that we need to make a decision on this right this minute.  If we decide that we want to write up templates and generate proper help separate from the man pages, then fine.  But I think that will be in addition to generating man pages for each of the commands.  We would probably want to do both.  

Here is what I suggest.  Let's go ahead and merge this and use it as a temporary measure until we can write templates for all of the help commands.  

BTW, I'm actually not crazy about using templates because they seem overly complicated and then we are back in the business of maintaining help docs even though we are using cobra.  But I'm fine using them if we really want to have help that is separate from the man pages.